### ISSUE TITLE
automatic movement after fight.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#258]
```
There seems to be problems with the new hold to move code:
http://andors.techby2guys.com/viewtopic.php?p=6241#p6241

After a fight the player moves onwards but there is no new touch input.
The game seems to remember the last position hold before the fight.
```

Original issue reported on code.google.com by `SamuelPl...@gmail.com` on 20 Oct 2011 at 8:14

**Comment by GoogleCodeExporter** [comment]
```
I think it can be closed.
```

Original comment by `SamuelPl...@gmail.com` on 21 Oct 2011 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
Resolved in r203.
```

Original comment by `oskar.wi...@gmail.com` on 14 Nov 2011 at 8:08
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
